.gitignore

```diff
@@ -1,2 +1,13 @@
-package-lock.json
-node_modules
+node_modules/
+.expo/
+npm-debug.*
```

Comment:
I love this asterisk syntax! I hadn't seen it before now, but it looks super useful